### ISSUE TITLE
Blockquotes in italic

### ISSUE THREAD

**Issue by NerosTie** [kalvn/Shaarli-Material#26]
With the new version of Shaarli, there is a plugin for markdown. I don't see any major issue but blockquotes could be better like being in italic.  
Blockquotes are already in a greyish font, so maybe it's not necessary.

What do you think about it?

Edit: I think there is a missing margin on the left when the markdown plugin is activated.

**Comment by kalvn** [comment]
I didn't test it yet but I'll have a look to improve styles on this part.

**Comment by kalvn** [comment]
As the content was purely text, I used to wrap it into a `<p>` tag. Now that there is the markdown plugin, I changed it to a div which fixes major issues. I let you have a look to the last version available in the master branch.

Concerning the blockquotes style, I prefer keeping things as-is as the markdown plugin embeds already a css file. Feel free to open an issue on the [official Shaarli repo](https://github.com/shaarli/shaarli) to request the blockquote to be in italic.

Feel free to close the ticket if you think the issue is solved.
